src/modules/auth/controllers/auth.controller.ts

```
@@ -1,33 +1,53 @@
-
 import { AuthService } from '../services/auth.service';
 import { RegisterDto } from '../dto/register.dto';
 import { LoginDto } from '../dto/login.dto';
+import { Request } from 'express';
```

ESLint

[error] 'Request' is defined but never used.

(@typescript-eslint/no-unused-vars)